nnvm/python/nnvm/testing/check_computation.py

```diff
@@ -121,7 +121,7 @@ def infer_shapes_dtypes(graph, shape=None, dtype=None, fallback_dtype=None):
         if input_shapes.get(x_name, x_shape) != x_shape:
             raise RuntimeError("Inferred shape differs from the provided shape.\n"
                                "Provided shapes: {}\nInferred shapes: {}"
-                               .format(shapes, input_shapes))
+                               .format(shape, input_shapes))
```

Comment:
Could you also help to rename these `shape`, `shapes`, `dtype`, `dtypes` stuffs explicitly based on their usage, e.g., `inferred_shapes`?

Reply:
Sure